web-wallet/src/routes/(app)/settings/+page.svelte

```diff
@@ -137,22 +138,22 @@
 			<div class="settings-group__multi-control-content">
 				<GasControls
 					on:gasSettings={(event) => {
-						if (event.detail.isValidGas) {
+						isGasValid = areValidGasSettings(event.detail.price, event.detail.limit);
+
+						if (isGasValid) {
 							settingsStore.update(store => {
 								store.gasLimit = event.detail.limit;
 								store.gasPrice = event.detail.price;
 
 								return store;
 							});
 						}
-
-						isValidGas = event.detail.isValidGas;
 					}}
 					limit={gasLimit}
-					limitLower={gasLimitLower}
-					limitUpper={gasLimitUpper}
+					limitLower={limitLower}
+					limitUpper={limitUpper}
 					price={gasPrice}
-					priceLower={gasPriceLower}
+					priceLower={priceLower}
```

Comment:
I would use the shorthand syntax, as now the prop name and the variable share the same name. Eg:

```
{limitLower}
{limitUpper}
price={gasPrice}
{priceLower}
```

I'm also seeing some discrepancy between the name choices for the variables – for example, `gasLimit` vs `limitLower`. Wouldn't it be better for all gas-related properties to follow consistent naming, such as `gasLimitLower`. It's a similar situation with the `gasPrice` and `priceLower`.